readme.md

```diff
@@ -390,7 +418,7 @@ import rateLimit, {
 } from 'express-rate-limit'
 
 /**
- * A {@link Store} that stores the hit count for each client.
+ * A `Store` that stores the hit count for each client.
```

Comment:
I wonder if we should move this example Store to a wiki or a separate file or something. I feel like it takes up a lot of space and it's not very relevant to the vast majority of the people who use this library. What do you think?

Reply:
I agree, it does feel a little out of place in the readme. We could move this to a `Guides` section of the wiki, in a page `Implementing your own store`.

Reply:
Yeah, I like that. And we can link to it from here.

Reply:
Ok, I'll do that then 👍🏾

Reply:
Hmm..  I don't see the `Wiki` tab on Github - could check if wikis are enabled for this repository?

Reply:
Oh, whoops. It's enabled now, and you should have edit permissions.

Reply:
Thanks <3